.github/workflows/release.yml

```diff
@@ -0,0 +1,103 @@
+name: Build Releases
```

Comment:
Looks like merge conflicts in here -- I'll look it over once those are resolved.

Reply:
Merge conflicts still need resolving, even after branch change.

Reply:
Yeah, the merge conflict is due to `develop` removing build-on-branches and this branch adding it back so I can test it on every push. I didn't plan to resolve until all testing was complete, since it's a nice reminder to delete that line and prevents us from accidentally keeping it. I can resolve if you prefer though, there are other ways to make sure it doesn't get merged (e.g. a Request Changes review on GH).

Reply:
Ok, I'll just review by diffing in IntelliJ against this file, but I'll hold off on doing the final approval and merge until the conflict is resolved, because I think it's a good idea to have another pair of eyes when resolving merge conflicts.